modules/orca/main.tf

```diff
@@ -29,6 +29,8 @@ module "orca_lambdas" {
   ## --------------------------
   ## REQUIRED
   orca_default_bucket                = var.orca_default_bucket
+  orca_sqs_metadata_queue_arn        = module.orca_sqs.orca_sqs_metadata_queue_arn
```

Comment:
Based on your changes to the `lambda/main.tf` file, you would not need the lines
```
orca_sqs_metadata_queue_arn = module.orca_sqs.orca_sqs_metadata_queue_arn
orca_sqs_metadata_queue_id = module.orca_sqs.orca_sqs_metadata_queue_id
```
but a line would need to be added
```
orca_sqs_metadata_queue_url = module.orca_sqs.orca_sqs_metadata_queue_id
```

Reply:
Fixed typo. Should make more sense now.